charts/kong/values.yaml

```diff
@@ -135,10 +135,16 @@ proxy:
 # The `name` property refers to the name of the ConfigMap or Secret
 # itself, while the pluginName refers to the name of the plugin as it appears
 # in Kong.
+# Subdirectories require separate ConfigMaps/Secrets. "path" indicates their
+# directory under the main plugin directory: the example below will mount the
+# contents of kong-plugin-rewriter-migrations at "/opt/kong/rewriter/migrations".
 plugins: {}
   # configMaps:
   # - pluginName: rewriter
   #   name: kong-plugin-rewriter
+  #   subdirectories:
```

Comment:
There is no mention anywhere in the comments here that the `subdirectories` values are optional and it is okay for plugins to not have a sub-directory at all, which is the case for majority of the plugins. Please add that bit of info so that users are not left confused.